### PR TITLE
Enroll auth service and API to the developer-portal

### PR DESCRIPTION
**Pull request by sldblog**
The operations engineering team started https://developer-portal.apps.live.cloud-platform.service.justice.gov.uk.

This change enrolls the system, service, database, API to the portal so it can be joined up by other services and APIs.

The definition is partially a copy from `hmpps-catalog-info.yaml` and [the definition in hmpps-architecture-as-code][arch-as-code].

[arch-as-code]: https://github.com/ministryofjustice/hmpps-architecture-as-code/blob/de0c1aa9fdf17c05b9c04112ec2237df38e0dd30/src/main/kotlin/uk/gov/justice/hmpps/architecture/model/HMPPSAuth.kt

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  type: service
+  lifecycle: production
+  system: system:hmpps-auth
+  owner: group:dps-tech
+  providesApis:
+    - api:hmpps-auth
+  consumesApis:
+    - api:hmpps-community
+  dependsOn:
+    - resource:hmpps-auth-database
```
Comment (sldblog) [inline catalog-info.yaml:37]: plus others, when they get defined (DOM1 ActiveDirectory? NOMIS DB/API?)

**Review by Mjwillis — APPROVED** [review]
LGTM

**Comment by sldblog** [comment]
Thank you 🙇‍♂️

**Comment by sldblog** [comment]
@Mjwillis I can't seem to merge, could you do it on behalf of me, please?

**Pull request merged**